infer/src/base/Config.ml

```diff
@@ -2094,6 +2094,12 @@ and pulse_cut_to_one_path_procedures_pattern =
      large procedures to prevent too-big states from being produced."
 
 
+and pulse_increase_leak_recall =
+    CLOpt.mk_bool ~default:false ~long:"pulse-increase-leak-recall"
+    {|Disable a check for potential false alarms in the memory leak checker, which also detects resource leaks.
+This is the has the effect of increasing the memory leak and resource leak recall.|}
```

Comment:
I think it would be better to have a more specific name tied to what the option does as in the future we could want to add more options that have the same effect of increasing recall but in other ways. `--pulse-pointer-arithmetic-can-suppress-leaks` (set to `true` by default: this option would mean the *opposite* of your current option as it's easier to explain that way around).